CONTRIBUTORS.md

```diff
@@ -0,0 +1,77 @@
+<html>

```

Comment:
@mbad0la 
Dear sir,
As far as I can see,  the design does conform to the all-contributor specs. Can you please pin point the errors that you find in the pull request?